DependencyInjection/Configuration.php

```diff
@@ -108,4 +109,95 @@ protected function configureClients(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    /**
+     * @param ArrayNodeDefinition $root
+     */
+    protected function configurePlugins(ArrayNodeDefinition $root)
+    {
+        $root->children()
+            ->arrayNode('plugins')
+                ->addDefaultsIfNotSet()
+                ->children()
+
+                    ->arrayNode('authentication')
+                    ->canBeEnabled()
+                        ->children()
+                            ->scalarNode('authentication')->isRequired()->cannotBeEmpty()->end()
```

Comment:
can we add an info to this? this is the name of the authentication service that implements some php-http interface?